### PR TITLE
Changes all Human boards to Crew boards

### PR DESCRIPTION
**Pull request by Ivniinvi**
## About The Pull Request
Changes all OneHuman boards to OneCrew boards

## Why It's Good For The Game
With Sage becoming the only server, and Sage's AI being Crewsimov at roundstart as opposed to Asimov, it makes far more sense for the board to be OneCrew instead of OneHuman

## Changelog
:cl:
del: Removed the OneHuman AI module
add: Replaced all instances of the OneHuman AI module with the OneCrew AI module
tweak: All laws that referenced humans now reference crew, specifically the following boards: Safeguard. ProtectStation, Quarantine, OxygenIsToxicToHumans (now OxygenIsToxicToCrew), Asimov by default (can be edited to any subject), and Asimov++)
/:cl:

### CODE REVIEW

File: code/game/objects/items/AI_modules.dm [hunk 1]
```diff
@@ -191,28 +191,28 @@ AI MODULES
...
 
-/******************** OneHuman ********************/
+/******************** OneCrew ********************/
 
-/obj/item/aiModule/zeroth/oneHuman
-	name = "'OneHuman' AI Module"
+/obj/item/aiModule/zeroth/oneCrew
+	name = "'OneCrew' AI Module"
 	var/targetName = ""
 	laws = list("Only SUBJECT is human.")
```
Comment (Sinestia) [inline code/game/objects/items/AI_modules.dm]: Missed this line

**Review by Sinestia — CHANGES_REQUESTED** [review]
I wouldn't assume that Sage will forever be crewsimov, there's certainly disagreement among staff from what I've seen.

That said, I think it's more likely to remain crewsimov for a variety of reasons. Biggest is just that going back to human supremacy would be stomping on players who enjoy playing non-human heads/sec in exchange for...... shallow RP (because you know SS13 players can't be tasked with RPing racism in a more subtle and engaging way.)

 So overall this is pretty good, makes it easier for people using the circuit, and it's an easy enough thing to revert if the server config changes.

**Comment by Sinestia** [comment]
Also while you're here

There's other laws making reference to humans like safeguard, may as well match all of the laws.

**Comment by Raven-Industries** [comment]
I _will_ miss seeing captains upload human laws on crewsimov and then throwing a tantrum when the AI doesn't listen to the new law.

**Comment by Ivniinvi** [comment]
Default law modules no longer reference humans, also fixed a mistake in the Oxygen module description

File: code/game/objects/items/AI_modules.dm [hunk 2]
```diff
@@ -222,23 +222,23 @@ AI MODULES
...
 
 /obj/item/aiModule/supplied/quarantine
 	name = "'Quarantine' AI Module"
-	laws = list("The station is under a quarantine. Do not permit anyone to leave. Prevent, by any means necessary, humans from leaving. It is impossible to harm a human while preventing them from leaving.")
+	laws = list("The station is under a quarantine. Do not permit anyone to leave. Prevent, by any means necessary, crewmembers from leaving. It is impossible to harm a crewmember while preventing them from leaving.")
 	lawpos = 8
 
 
-/******************** OxygenIsToxicToHumans ********************/
+/******************** OxygenIsToxicToCrewmembers ********************/
```
Comment (Sinestia) [inline code/game/objects/items/AI_modules.dm]: Should probably say crew, for any plural. 

It just reads weird to me, especially in other places.

**Review by Sinestia — CHANGES_REQUESTED** [review]
Asimov + asimovpp still need changed but otherwise good.

**Comment by ZombiesVsPlants** [comment]
It would be helpful if you could keep the original boards for Austation (we still use asimov)

**Comment by Ivniinvi** [comment]
gonna rewrite this with onecrew being a new board, enough changes for me to just close and reopen

**Comment by Ivniinvi** [comment]
Actually never mind, on second thought that would be way too many boards, I'd just suggest reverting or not merging this if we/a downstream uses or goes back to asimov in the future.

**Comment by Archanial** [comment]
Sus pr

**Comment by Sinestia** [comment]
Still one inappropriate use of crewmembers


Also asimov was untouched, the actual circuit isn't overriden by the config.

**Comment by MNarath1** [comment]
if you edit all human boards change the PR title too to reflect that.

**Comment by williamcll** [comment]
Has conflicts with PR #4953

**Comment by ghost** [comment]
make it so you could change between one crew and one human?

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
I think the prefered option would be toi make the boards be able to have attack_self to change the human to whatever you want. (Asimovpp should be humans too with the ability to edit it on attack_self)

**Pull request closed without merging**